src/api/dnaApi.ts

```diff
@@ -22,6 +28,14 @@ export async function hentSyfoVirksomheter(): Promise<Organisasjon[]> {
     throw new Error('Feil ved kontakt mot baksystem.');
 }
 
+export async function hentRefusjonstatus(): Promise<RefusjonStatus[]> {
+    const respons = await fetch(refusjonstatusURL);
+    if (respons.ok) {
+        return await respons.json();
```

Comment:
kunne vurdert å bruke zod for at frontenden parser og sier i fra hvis svaret har feil format.